suite-native/intl/src/en.ts

```diff
@@ -1005,6 +1005,23 @@ export const en = {
                 standard: 'Standard wallet',
                 defaultPassphrase: 'Passphrase wallet #{index}',
             },
+            autoEject: {
+                switch: {
+                    title: 'Auto eject',
+                    description: 'Eject all wallets automatically after unplugging Trezor',
+                    alert: {
+                        titleNoConnectedTrezor: 'Enabling auto eject will eject all wallets',
+                        titleConnectedTrezor: ' after disconnecting Trezor',
```

Comment:
This does not seem to be right or?

Reply:
`titleConnectedTrezor` is only used for specific case and is concatanated with the previous translation string - look for the usage.

However I guess it's unreadable and I will just create 2 separate strings.